guides/v2.1/mrg/ee/MsrpStaging.md

```diff
@@ -23,4 +23,4 @@ You can find the list of modules that have dependencies on the Magento_MsrpStagi
 
 ## Additional information
 
-For more Magento 2 developer documentation, see [Magento 2 Developer Documentation]({{ site.baseurl }}). Also, there you can track [backward incompatible changes made in a {{site.data.var.ee}} mainline after the Magento 2.0 release]({{ site.baseurl }}/guides/v2.0/release-notes/backward-incompatible-changes/commerce.html).
+For more Magento 2 developer documentation, see [Magento 2 Developer Documentation]({{ site.baseurl }}/). Also, there you can track [backward incompatible changes made in a {{site.data.var.ee}} mainline after the Magento 2.0 release]({{ site.baseurl }}/guides/v2.0/release-notes/backward-incompatible-changes/commerce.html).
```

Comment:
I'm glad you fixed these links, but it's not a good idea to mix too many different types of commits in a single PR. It's difficult to understand the PRs purpose after a while.

Reply:
I understand this and totally agree.
But I had to ensure that errors reported by the html-proofer on staging were caused by broken links, not bugs in my script.

Reply:
I can revert the commit with broken links fix in this PR

Reply:
No, don't worry about reverting.